src/core/ddsc/include/dds/ddsc/dds_public_status.h

```diff
@@ -335,7 +335,7 @@ dds_get_offered_deadline_missed_status(
  */
 DDS_EXPORT dds_return_t
 dds_get_offered_incompatible_qos_status (
-  dds_entity_t writer,
```

Comment:
This really only takes a writer, so I think "writer" is the better choice.

Reply:
Applied and fixed.